master/README.md

```diff
@@ -1,16 +1,10 @@
-# Velero
+![](/docs/img/velero.png)
```

Comment:
I *think* this needs to manually be changed to just be (`/img/velero.png`), since the `docs/` dir is stripped off in the `gh-pages` branch.